src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -646,7 +646,14 @@ protected async Task CheckoutSettingsFileAsync()
         /// file with the name LaunchSettings.json. We don't need to special case because, if a file with this name
         /// changes we will only check if the one we cared about was modified.
         /// </summary>
-        protected void LaunchSettingsFile_Changed(object sender, FileSystemEventArgs e)
+        private void LaunchSettingsFile_Changed(object sender, FileSystemEventArgs e)
+        {
+#pragma warning disable VSTHRD110 // Observe result of async calls
+            HandleLaunchSettingsFileChangedAsync();
```

Comment:
.Forgot is the pattern.

Reply:
Updated.

We have a few other unobserved tasks throughout the code (search for `nowait` for example). Are these candidates for `.Forget()` as well? They don't raise `VSTHRD110`.

It seems that `.Forget()` doesn't actually do anything with the tasks beyond suppress warnings. Is that really a great idea? Can we do something more sensible with them that's specific to our code, such as provide a continuation in case of fault that logs something sensible, or launches a debugger in `DEBUG` builds, or...